pkg/backend/filestate/backend.go

```diff
@@ -487,6 +498,13 @@ func (b *localBackend) apply(
 	return changes, nil
 }
 
+// query executes a query program against the resource outputs of a locally hosted stack.
+func (b *localBackend) query(ctx context.Context, stack backend.Stack, op backend.UpdateOperation,
+	events chan<- engine.Event) result.Result {
+
+	return result.Error("Local backend does not support querying over the state")
```

Comment:
This is something we will support, right?  Could you mark this with a TODO?  I expect that the implementation will be very close to what we do for http backed stacks?

Reply:
We can add a TODO. It would be pretty easy to do, I didn't because I wasn't sure we wanted to.